frontend/components/PhoneinputTest.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { Component, useRef } from "react";
+import {
+  SafeAreaView,
+  StyleSheet,
+  View,
+  StatusBar,
+  TouchableOpacity,
+  Text,
+  Pressable,
+} from 'react-native';
+import PhoneInput from 'react-native-phone-number-input';
+// import {Colors} from 'react-native/Libraries/NewAppScreen';
+// const phoneInput = useRef<PhoneInput>(null);
+type State = {
+    value: string,
+    countryCode: string,
+    formattedValue: string,
+    valid:boolean,
+    disabled: boolean,
+    showMessage: boolean,
+
+}
+class PhoneinputTest extends Component<{}, State> {
+    phoneInput=React.createRef<PhoneInput>();
+    constructor(props: {}) {
+      super(props);
+      this.state = {
+        value: '',
+        countryCode: '',
+        formattedValue: '',
+        valid: false,
+        disabled: false,
+        showMessage: false,
+      }
+    }
+  demo = () =>{
+      console.log("demo triggered")
+  }
+  
+  render(){
+    return (
+        <>
+          <StatusBar barStyle="dark-content" />
+          <View style={styles.container}>
+            <SafeAreaView style={styles.wrapper}>
+              {this.state.showMessage && (
+                <View style={styles.message}>
+                  <Text>Country Code : {this.state.countryCode}</Text>
```

Comment:
I see why it is called 'Test'...cleanup this code if not needed?

Reply:
ok